### PR TITLE
test(data_structures): format test names

### PR DESCRIPTION
**Pull request by timreichen**
Ref: https://github.com/denoland/deno_std/issues/3754

### CODE REVIEW

File: data_structures/binary_heap_test.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ Deno.test("[collections/BinaryHeap] with default descend comparator", () => {
   assertEquals(actual, expected);
 });
 
-Deno.test("[collections/BinaryHeap] with ascend comparator", () => {
+Deno.test("BinaryHeap() checks ascend comparator", () => {
```
Comment (iuioiua) [inline data_structures/binary_heap_test.ts:44]:
```suggestion
Deno.test("BinaryHeap() works with ascend comparator", () => {
```

File: data_structures/binary_heap_test.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { BinaryHeap } from "./binary_heap.ts";
 import { ascend, descend } from "./comparators.ts";
 import { Container, MyMath } from "./_test_utils.ts";
 
-Deno.test("[collections/BinaryHeap] with default descend comparator", () => {
+Deno.test("BinaryHeap() checks default descend comparator", () => {
```
Comment (iuioiua) [inline data_structures/binary_heap_test.ts:7]:
```suggestion
Deno.test("BinaryHeap() works with default descend comparator", () => {
```

**Review by iuioiua — CHANGES_REQUESTED** [review]
Again, the use of the word "checks" doesn't make much sense to me. Are you able to sweep through and tweak?

**Review by iuioiua — APPROVED** [review]
I've made some adjustments. Now, LGTM. Thank you.

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**